tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -50,6 +52,13 @@ def montecarlo_radial1d(model, plasma, runner):
         runner.j_blue_estimator,
         runner.Edotlu_estimator,
     )
+
+    # Configuring the Tracking for R_Packets
+    if montecarlo_configuration.RPACKET_TRACKING:
+        tracked_rpacket = RPacketCollection()
```

Comment:
Perhaps `tracked_rpacket` should be `tracked_rpackets` because there is more than 1 packet being tracked in this variable

Reply:
Moved to another PR :)